docs/tools/vdb_table/data/databricks.json

```diff
@@ -0,0 +1,188 @@
+{
+    "name": "Databricks Vector Search",
+    "links": {
+      "docs": "https://docs.databricks.com/en/generative-ai/vector-search.html",
+      "github": "",
+      "website": "https://www.databricks.com/product/machine-learning/vector-search",
+      "vendor_discussion": "https://github.com/superlinked/VectorHub/discussions/514",
+      "poc_github": "kobindra",
+      "slug": "databricks"
+    },
+    "oss": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "license": {
+      "value": "Proprietary",
+      "source_url": "https://www.databricks.com/legal/mcsa",
+      "comment": ""
+    },
+    "dev_languages": {
+      "value": [
+        ""
+      ],
+      "source_url": "",
+      "comment": ""
+    },
+    "vector_launch_year": 2024,
+    "metadata_filter": {
+      "support": "full",
+      "source_url": "https://www.databricks.com/blog/announcing-mosaic-ai-vector-search-general-availability-databricks",
+      "comment": ""
+    },
+    "hybrid_search": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html",
+      "comment": ""
+    },
+    "facets": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "geo_search": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "multi_vec": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "sparse_vectors": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "bm25": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html#how-does-mosaic-ai-vector-search-work",
+      "comment": "Okapi BM25"
+    },
+    "full_text": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html",
+      "comment": ""
+    },
+    "embeddings_text": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html#options-for-providing-vector-embeddings",
+      "comment": ""
+    },
+    "embeddings_image": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html#options-for-providing-vector-embeddings",
+      "comment": "With self-managed Delta Sync"
+    },
```

Comment:
I don't see any image models in the catalog linked above. Can you say if there is?